alicloud/provider.go

```diff
@@ -308,6 +309,7 @@ func Provider() terraform.ResourceProvider {
 			"alicloud_cs_swarm":                            resourceAlicloudCSSwarm(),
 			"alicloud_cs_kubernetes":                       resourceAlicloudCSKubernetes(),
 			"alicloud_cs_managed_kubernetes":               resourceAlicloudCSManagedKubernetes(),
+			"alicloud_cs_serveless_kubernetes":             resourceAlicloudCSServelessKubernetes(),
```

Comment:
Can you confirm the name is `alicloud_cs_serveless_kubernetes` not `alicloud_cs_serverless_kubernetes `?

Reply:
you are right. The correct name is serverless